plugin/proto/tflint.proto

```diff
@@ -180,8 +180,7 @@ message EvaluateExpr {
     }
 
     message Request {
-        bytes expr = 1 [deprecated = true];
-        Range expr_range = 2 [deprecated = true];
+        reserved 1, 2;
```

Comment:
These fields are deprecated in https://github.com/terraform-linters/tflint-plugin-sdk/pull/205. SDK v0.14+ and TFLint v0.42+ do not use them.
Support for older SDK versions has already been dropped in https://github.com/terraform-linters/tflint/pull/1749.

On the other hand, note that `IncludeNotCreated`, which was deprecated in https://github.com/terraform-linters/tflint-plugin-sdk/pull/208, cannot be dropped yet.
This is exposed directly to plugin developers and must be unpublished first.